get_number.rb

```diff
@@ -0,0 +1,48 @@
+# Get My Number Game
+# Written by: you!
+
+puts "Welcome to 'Get My Number!'"
+
+# Получение имени игрока и вывод приветствия.
+print "What's your name? "
+input = gets
+name = input.chomp
+puts "Welcome, #{name}!"
+
+# Сохранение случайного числа.
+puts "I've got a random number between 1 and 100."
+puts "Can you guess it?"
+target = rand(100) + 1
```

Comment:
rubocop recommends using ranges instead of offsets:
```
rand(start..end)

Reply:
The error is still here:
```
get_number.rb:15:10: C: Style/RandomWithOffset: Prefer ranges when generating random numbers instead of integers with offsets.
target = rand(100) + 1
         ^^^^^^^^^^^^^
```

Reply:
ok